From fc28220052a6043a04e65ea0f89239779a660a78 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:23:02 -0400
Subject: [redhat] kabi: Adding symbol dma_ops (arch/x86/kernel/pci-dma.c)

Message-id: <497d7dba6af7d642ef8a437097edf41c6585f881.1410510773.git.poros@redhat.com>
Patchwork-id: 96028
O-Subject: [PATCH RHEL7.1 062/233] kabi: Adding symbol dma_ops (arch/x86/kernel/pci-dma.c)
Bugzilla: 1071098
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1071098
    https://bugzilla.redhat.com/show_bug.cgi?id=1071098

    Description:
    ============
    LLT RDMA uses ib_dma_(un)map_single, which uses this.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/dma_ops

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_x86_64/dma_ops b/redhat/kabi/kabi-module/kabi_x86_64/dma_ops
new file mode 100644
index 0000000..6214718
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/dma_ops
@@ -0,0 +1,2 @@
+#1-
+0x584c5b17 dma_ops vmlinux EXPORT_SYMBOL
-- 
1.7.1